sktime/forecasting/darts.py

```diff
@@ -0,0 +1,665 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Interfaces to estimators from darts by Unit8."""
+
+from typing import Optional
+
+from sktime.forecasting.base.adapters._darts import (
+    FUTURE_LAGS_TYPE,
+    LAGS_TYPE,
+    PAST_LAGS_TYPE,
+    _DartsRegressionAdapter,
+    _DartsRegressionModelsAdapter,
+)
+from sktime.utils.warnings import warn
+
+__author__ = ["yarnabrina", "fnhirwa"]
+
+
+class DartsRegressionModel(_DartsRegressionAdapter):
+    """Darts Regression Model Estimator.
+
+    Parameters
+    ----------
+    lags
```

Comment:
can you make sure to adopt the docstring style with type and defaults?

Reply:
yeah sure.